### ISSUE TITLE
Don't show Cree words labels in spoken recordings, unless normatized

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#149]
When implementing #128, the old itwêwina shows the search string that is used in order to access spoken recordings. This search string appears to be without vowel-length, which if shown to a user is a non-standard spelling of the word in question.

E.g. `wapamew` for `wâpamêw`

Thus, until/unless the search strings can be presented in an orthographically standardized form, do not show them in the new itwêwina. Rather, show just a microphone symbol and the name and community of the speaker. Something like `(≪ Harley Simon, Maskwacîs`.

Thus, itwêwina would not be generating or showing any non-standard Cree (in SRO), and someone in Regina will be happy.

Desired behavior: `wâpamêw`, or `Ø`
Undesired behavior: `wapamew`

Of course, for recordings of inflected forms that we present alongside the lemma (like `ê-wâpamât`) we would need to show the normatized spelling, so as not to confuse users.

**Comment by eddieantonio** [comment]
tl;dr: either we show it RIGHT or we don't show it at all!

(suggested by Arok)

**Comment by Madoshakalaka** [comment]
I think this is already achieved. Verify please @aarppe ?

**Comment by aarppe** [comment]
I don't know if this is implemented in the new dictionary, as I don't see any recordings yet, e.g.

https://sapir.artsrn.ualberta.ca/cree-dictionary/search/wapamew/

In the old itwêwina, we certainly yet show the non-standard transcriptions of the recordings (based on how the search from the speech database is done), cf.

https://altlab.ualberta.ca/itwewina/detail/crk/eng/w%C3%A2pam%C3%AAw.html?no_compounds=true&lemma_match=true&e_node=7116418259216475478

But we don't want to replicate this behavior in the new itwêwina (until we figure out a more elaborate design), so in the new itwêwina we should only show speaker data and not at all the transcription of the speech snippet.

**Comment by Madoshakalaka** [comment]
@aarppe sorry I didn't recognize this issue is talking about recordings
